spec/world_spec.rb

```diff
@@ -5,58 +5,100 @@
+      third_cell = Cell.new([1,1])
+      world = World.new([first_cell, second_cell, third_cell])
 
-    world.evolve
+      world.evolve
 
-    expect(world.cells).to eq([])
+      expect(world.cells).to eq([])
+    end
   end
 
-  it "returns how many live cells a position has around" do
-    first_cell = Cell.new([0,0])
-    second_cell = Cell.new([0,1])
-    third_cell = Cell.new([1,0])
-    world = World.new([first_cell, second_cell, third_cell])
+    it "returns how many live cells a position has around" do
+      first_cell = Cell.new([0,0])
+      second_cell = Cell.new([0,1])
+      third_cell = Cell.new([1,0])
+      world = World.new([first_cell, second_cell, third_cell])
+
+      neighbours_first_cell = world.count_live_neighbours([0,0])
+      neighbours_second_cell = world.count_live_neighbours([0,1])
+      neighbours_third_cell = world.count_live_neighbours([1,0])
+      neighbours_empty_position = world.count_live_neighbours([1,2])
+      neighbours_empty_position2 = world.count_live_neighbours([2,2])
+
+      expect(neighbours_first_cell).to eq(2)
+      expect(neighbours_second_cell).to eq(2)
+      expect(neighbours_third_cell).to eq(2)
+      expect(neighbours_empty_position).to eq(1)
+      expect(neighbours_empty_position2).to eq(0)
+    end
+
+  describe "applies the last rule concerning dead cells" do
+    it "resuscitates dead cells if they have exactly 3 live neighbours" do
+      first_cell = Cell.new([0,0])
+      second_cell = Cell.new([1,0])
+      third_cell = Cell.new([0,1])
+      fourth_cell = Cell.new([2,-1])
+      fifth_cell = Cell.new([-1,-1])
+      world = World.new([first_cell, second_cell, third_cell, fourth_cell, fifth_cell])
+      expect(world.count_live_neighbours([1,1])).to eq(3)
+      expect(world.count_live_neighbours([-1,0])).to eq(3)
+      expect(world.count_live_neighbours([0,-1])).to eq(3)
+      expect(world.count_live_neighbours([1,-1])).to eq(3)
+
+      world.evolve
+
+      expect(world.cells_positions).to contain_exactly([0,0], [1,0], [0,1], [1,1], [1,-1],[0,-1],[-1,0])
+    end
+
+    it "returns positions around live cells where new cells could resuscitate" do
+      first_cell = Cell.new([0,0])
+      second_cell = Cell.new([0,1])
+      third_cell = Cell.new([1,1])
+      world = World.new([first_cell, second_cell, third_cell])
+
+      potential_resuscitating_cells = world.positions_of_potential_resuscitating_cells
+
+      expect(potential_resuscitating_cells).to contain_exactly([-1,1],[-1,2],[0,2],[1,2],[1,0],[1,-1],[0,-1],[-1,-1],[-1,0],[2,2],[2,1],[2,0])
+    end
+  end
 
-    neighbours_first_cell = world.count_live_neighbours([0,0])
-    neighbours_second_cell = world.count_live_neighbours([0,1])
-    neighbours_third_cell = world.count_live_neighbours([1,0])
-    neighbours_empty_position = world.count_live_neighbours([1,2])
-    neighbours_empty_position2 = world.count_live_neighbours([2,2])
+    it "returns positions of all live cells" do
```

Comment:
This one is good too. But I would probably put it somewhere higher up. It seems like this test would come first and once you have the method `cells_positions` nailed down, then you can build on it and get the other methods working.